models/follow.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Follow = sequelize.define('Follow', {
+    follower: DataTypes.STRING,
+    following: DataTypes.STRING
+  }, {});
+  Follow.associate = function(models) {
+    // associations can be defined here
+    Follow.belongsTo(models.User, {
+      as: 'followKey',
+      foreignKey: 'follower',
+      onDelete: 'CASCADE'
+    });
+  };
+  return Follow;
+};
```

Comment:
hey, we removed the blank line here and in some other places. 
You could consider configuring your editor to automatically format these files.
or you can set up a pre-commit hook to clean the new files before they get pushed.